bridges/ethereum/src/eth.rs

```diff
@@ -123,6 +129,20 @@ impl WrbRequests {
             }
         }
     }
+    /// Update a data request in `Posted` state. This means the claimer changed.
+    pub fn update_posted_claimer(&mut self, dr_id: U256, address: H160) {
+        if let Some(DrState::Posted { .. }) = self.requests.get(&dr_id) {
+            if let Some((x, _)) = self.posted.get_mut(&dr_id) {
+                *x = address;
+                self.requests.insert(dr_id, DrState::Posted { address });
+            } else {
+                log::warn!(
+                    "Cannot update posted state because dr is in an inconsistent state: [{}]",
```

Comment:
Should we move this warning to debug? If the DR is not in posted state, we don't need to know the poster address.

```suggestion
                log::debug!(
                    "Cannot update claimer address because dr is not in posted state: [{}]",
```

Reply:
makes sense